src/main/java/seedu/address/model/person/Phone.java

```diff
@@ -11,8 +11,8 @@
 
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Phone numbers should only contain numbers, and it should be at least 3 digits long";
-    public static final String VALIDATION_REGEX = "\\d{3,}";
+            "Phone numbers should only contain numbers, and it should be 3 to 15 digits long";
+    public static final String VALIDATION_REGEX = "\\d{3,15}";
```

Comment:
According to sources online, e.g. [link](https://stackoverflow.com/questions/723587/whats-the-longest-possible-worldwide-phone-number-i-should-consider-in-sql-varc). Maximum phone number is indeed 15. However, this is not inclusive of the prefix and suffix. If we add in prefix and suffix, the max should be 15 + 5 (prefix) + 11 (suffix) = 31.